Docker_cram/dockerfile

```diff
@@ -2,103 +2,60 @@
 FROM osrf/ros:noetic-desktop-full
 SHELL ["/bin/bash", "-c"]
 
-#source ROS
-RUN source /opt/ros/noetic/setup.bash
-
```

Comment:
Could you please remove these unwanted lines.

Reply:
It's done. the red indicators are the lines that got removed from your dockerfile